src/handlers/admin/menu_handler.py

```diff
@@ -242,79 +155,19 @@ def message_handler(update: Update, context: CallbackContext):
                         next_job_func, new_timeout, context=job_context
                     )
 
-            keyboard = [
-                [
-                    InlineKeyboardButton(
-                        "К настройке чата",
-                        callback_data=json.dumps(
-                            {
-                                "chat_id": chat_id,
-                                "action": constants.Actions.select_chat,
-                            }
-                        ),
-                    ),
-                    InlineKeyboardButton(
-                        "К списку чатов",
-                        callback_data=json.dumps(
-                            {"action": constants.Actions.start_select_chat}
-                        ),
-                    ),
-                ],
-            ]
-
-            reply_markup = InlineKeyboardMarkup(keyboard)
-            update.message.reply_text(
-                constants.on_success_set_kick_timeout_response,
-                reply_markup=reply_markup,
-            )
+            context.user_data["action"] = None
+            send_message_with_keyboard(update, constants.on_success_set_kick_timeout_response, user_chat_id)
 
         elif action in [
             constants.Actions.set_on_new_chat_member_message_response,
-            constants.Actions.set_notify_message,
-            constants.Actions.set_on_known_new_chat_member_message_response,
-            constants.Actions.set_on_successful_introducion_response,
-            constants.Actions.set_on_kick_message,
+            # ... other actions ...
         ]:
             message = update.message.text_markdown
-            with session_scope() as sess:
-                if action == constants.Actions.set_on_new_chat_member_message_response:
-                    chat = Chat(id=chat_id, on_new_chat_member_message=message)
-                if (
-                    action
-                    == constants.Actions.set_on_known_new_chat_member_message_response
-                ):
-                    chat = Chat(id=chat_id, on_known_new_chat_member_message=message)
-                if action == constants.Actions.set_on_successful_introducion_response:
-                    chat = Chat(id=chat_id, on_introduce_message=message)
-                if action == constants.Actions.set_notify_message:
-                    chat = Chat(id=chat_id, notify_message=message)
-                if action == constants.Actions.set_on_kick_message:
-                    chat = Chat(id=chat_id, on_kick_message=message)
-                sess.merge(chat)
+            attribute_map = {
+                constants.Actions.set_on_new_chat_member_message_response: "on_new_chat_member_message",
```

Comment:
yeah, this is the `callback_map` I wrote about above -- but it makes sense to make this a global dict (e.g. in the `__init__.py` module, so that it later could be imported easily in `app.py` and used to create `CallbackQueryHandler`

Reply:
I see, we also want to have a quick map between buttons (e.g. `set_on_new_chat_member_message_response` and DB fields).... maybe have a separate map for that?